### PR TITLE
Bump container image tag to fix incompatibility between the SDK and the container.

### PR DESCRIPTION
**Pull request by tvalentyn**
------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand:
   - [ ] What the pull request does
   - [ ] Why it does it
   - [ ] How it does it
   - [ ] Why this approach
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Comment by lukecwik** [comment]
Run Python Dataflow ValidatesRunner

File: sdks/python/apache_beam/runners/dataflow/internal/dependency.py [hunk 1]
```diff
@@ -77,7 +77,7 @@
 
 # Update this version to the next version whenever there is a change that will
 # require changes to legacy Dataflow worker execution environment.
-BEAM_CONTAINER_VERSION = 'beam-master-20171214'
+BEAM_CONTAINER_VERSION = 'beam-master-20180228'
 # Update this version to the next version whenever there is a change that
 # requires changes to SDK harness container or SDK harness launcher.
 BEAM_FNAPI_CONTAINER_VERSION = 'beam-master-20171222'
```
Comment (lukecwik) [inline sdks/python/apache_beam/runners/dataflow/internal/dependency.py]: Why not update this as well?

Comment (tvalentyn) [reply sdks/python/apache_beam/runners/dataflow/internal/dependency.py]: We don't have to unless there were changes to python-fnapi container image. Looks like this time there were changes container image, so it's a good idea to update.

**Comment by tvalentyn** [comment]
Run Python Dataflow ValidatesRunner

**Comment by tvalentyn** [comment]
Ready for merge.

**Pull request merged**